cli/tests/workers_test.ts

```diff
@@ -84,10 +98,13 @@ testPerm({ net: true }, async function workerCanUseFetch(): Promise<void> {
     promise.reject(e.message);
   };
 
+  // Defer promise.resolve() to allow worker to shut down
   fetchingWorker.onmessage = (e): void => {
     assert(e.data === "Done!");
     promise.resolve();
   };
 
   await promise;
 });
+
+await Deno.runTests();
```

Comment:
Is moving this workers_test related to the assertOps?

Reply:
Yes, workers are leaking ops at the moment because there is no way to terminate worker (not implemented yet), added FIXME at the top of the file